.travis.yml

```diff
@@ -27,7 +27,7 @@ script:
   - docker build --build-arg VCS_REF=`git rev-parse --short HEAD`   --build-arg BUILD_DATE=`date -u +”%Y-%m-%dT%H:%M:%SZ”` -t scoreucsc/bassa  .
 
 after_success:
-  - if [ "$TRAVIS_BRANCH" == "master" ]; then
+  - if [ "$TRAVIS_BRANCH" == "master" && "$TRAVIS_PULL_REQUEST" == "false" ]; then
```

Comment:
It seems the line should be `"$TRAVIS_BRANCH" = "master" && "$TRAVIS_PULL_REQUEST" = "false"`